x-pack/filebeat/module/misp/threat/config/input.yml

```diff
@@ -10,6 +10,7 @@ interval: {{ .interval }}
 json_objects_array: {{ .json_objects_array }}
 pagination: {{ .pagination }}
 url: {{ .url }}
+ssl: {{ .ssl }}
```

Comment:
It would be great if you can provide any test to verify correctness of settings.